src/main/java/xyz/nucleoid/plasmid/mixin/bubble/ServerWorldMixin.java

```diff
@@ -61,6 +66,13 @@ private ServerWorldMixin(MutableWorldProperties properties, RegistryKey<World> r
         super(properties, registryKey, dimension, profiler, client, debug, seed);
     }
 
+    @Inject(method = "<init>", at = @At("RETURN"))
```

Comment:
We should probably add a new mixin class for this in a different package - it should be fine to have multiple ServerWorldMixins.

Reply:
On second thought, this might not be necessary at all: I don't think it's strictly required that the `PersistentState` is initialized on world init

Reply:
Yes, a `PersistentState` doesn't have to be initialized on world init. 
It only reads the file when the `PersistentState` is first created. 
But where do you think it might be a good place to do it since it still needs the Overworld `ServerWorld`?.
A possible area I can think of would be `MinecraftServer#loadWorld()`

Reply:
The initialization logic can just be delegated to getOrCreate- the get function could go through the MinecraftServer to get the overworld instance

Reply:
Oh yeah sure, but where should I call the `get()` method?
It needs to happen sometime after `MinecraftServer#createWorlds()` so it has the Overworld.

Reply:
The `get()` function doesn't ever need to be called by plasmid- whatever code is making use of the storage can call it whenever it needs to and it can initialize then

Reply:
I suppose it could be made to work like that, but if that is how it'd work then mods would need to call `ServerStorageManager#get()` every time they plan on loading their `ServerStorage`(s) which could get repetitive and a bit ugly looking and would also require `MinecraftServer` access. 

The reason I wanted to initialize it when the Server is loading is so that it's cleaner and easier for mods to load their `ServerStorage`(s) since it would load all the `ServerData`s in a bunch.

Reply:
oh! duh, okay- i get it. sorry about that :P
this should be good to just have in a separate mixin file then 👍